docs/getting-started/styling-messages/markdown.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Markdown
+description: Markdown syntax usable to format text on Discord.
+---
+
+import {
+    DiscordInteraction,
+    DiscordMessage,
+    DiscordMessages,
+    DiscordEmbed,
+    DiscordEmbedField,
+    DiscordEmbedFields,
+    DiscordMarkdown,
+} from "discord-message-components/packages/react";
+import "discord-message-components/packages/react/dist/style.css";
+
+import DiscordComponent from "../../../src/components/DiscordComponent";
+
+# Markdown
+Markdown is a type of markup language that's limited in terms of formatting yet simple. Discord allows
+for a watered-down version of markdown to be in their messages.
+
+## Text Markdown
+
+Text formatting can be used in messages and in most embed parts,
+as explained in the dedicated section below.
+ 
+<table>
+    <tr>
+        <td>*italics*</td>
+        <td>__*underlined italics*__</td>
+    </tr>
+    <tr>
+        <td>**bold**</td>
+        <td>__**underlined bold**__</td>
+        
+    </tr>
+    <tr>
+        <td>***bold italics***</td>
+        <td>__***underlined bold italics***__</td>
+        
+    </tr>
+    <tr>
+        <td>__underlined__</td>
+        <td>~~strikethrough~~</td>
+    </tr>
+</table>
+
+## Code Markdown
+
+To create a single-line code block without syntax highlight, wrap the text between single backticks.
+This code block will only add a dark background to the text.
+
+```
+`print("Hello world!")`
+```
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+        `print("Hello world!")`
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+To create a multi-line code block without syntax highlight, wrap the text between triple backticks
+on first and last line. This type of code block will encase the code inside a box.
+
+````
+```
+message = "Hello world!"
+print(message)
+```
+````
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+```
+message = "Hello world!"
+print(message)
```

Comment:
Note: this doesn't format correctly at the moment. Fix before merging. May be resolved by #239.